<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWeatherData</class>
 <widget class="QWidget" name="QWeatherData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLineEdit" name="txtTemperature">
     <property name="text">
      <string>No temp. data</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtHumidity">
     <property name="text">
      <string>No humidity data</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="txtLight">
     <property name="text">
      <string>No light data</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="prgTemperature">
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QProgressBar" name="prgHumidity">
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="txtPressure">
     <property name="text">
      <string>No pressure data</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblTitle">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Weather station</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblTimestamp">
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lblStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Status: -</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="lblLightnings">
     <property name="text">
      <string>Lightnings: -</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionShowCurrent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Current readings</string>
   </property>
   <property name="toolTip">
    <string>Show the current readings</string>
   </property>
  </action>
  <action name="actionShowSmoothed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Smoothed</string>
   </property>
   <property name="toolTip">
    <string>Show the smoother current readings</string>
   </property>
  </action>
  <action name="actionShowAverage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Average</string>
   </property>
   <property name="toolTip">
    <string>Show long term average</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear current readings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
